main/commons/src/main/java/org/cryptomator/common/vaults/Volume.java

```diff
@@ -86,9 +88,12 @@ public VolumeException(String message, Throwable cause) {
 	}
 
 	/**
-	 * Interface to bundle the different revealer interfaces in the used nio-adapters
+	 * Hides and unifies the different Revealer implementations in the different nio-adapters.
 	 */
-	interface Revealer extends org.cryptomator.frontend.fuse.mount.Revealer, org.cryptomator.frontend.dokany.Revealer, Mounter.Revealer{
+	@FunctionalInterface
+	interface RevealerFacade {
```

Comment:
Confusing name, since this is not a Facade in the strict sense: In this case, there is no "implementation" that gets hidden behind a facade, since all three hidden Revealers just serve the purpose to inject an caller-provided implementation.

By my misleading last review comment asking to "hide the implementation", I meant not to expose their class names to whatever object is providing a Revealer. This has now been solved, though.

Still I guess we should either just call it `Revealer` or, if you think thats too confusing, stay in line with the Volume classes and call it `VolumeRevealer`.